bom/application/pom.xml

```diff
@@ -5608,6 +5608,17 @@
                 <artifactId>quarkus-devtools-registry-client</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-devtools-testing</artifactId>
+                <scope>test</scope>
```

Comment:
We usually don't define the scope in the BOM as it's bad practice.

Reply:
Ok, I'll remove it.